index.html

```diff
@@ -0,0 +1,390 @@
+            3
+            <i
+              class="fa fa-share"
+              style="cursor: pointer; margin-left: 10px; margin-right: 10px;"
+              aria-hidden="true"
+            ></i>
+            share
+            <i
+              class="material-icons"
+              style="cursor: pointer; margin-left: 10px; margin-right: 10px;"
+              >more_horiz</i
+            >
+          </span>
+        </div>
+        <div style="display: flex;">
+          <div>
+
+                               <!-- Channel Info -->
+
+            <img
+              style="display: inline-block;"
+              id="channel-icon"
+              alt=""
+              src="microverse-logo.png"
+            />
+            <h4 style="display: inline-block;" id="channel-name">Microverse</h4>
+          </div>
+
+          <button id="subscribe">Subscribe</button>
+        </div>
+
+
+                            <!-- Video Description -->
+
+
+        <h4 id="description">
+          Microverse is an online school for remote software developers where
+          you pay nothing until you land a life-changing job, no matter where
+          you live.<br><br>
+
+          At Microverse, there are no classes or physical classrooms. Instead,
+          students learn through a very unique approach — by remote pair
+          programming and collaborating with other students in real-time, just
+          as you would with colleagues in a real company.<br /><br />
+        </h4>
+      </section>
+
+
+                                 <!-- Suggested Videos -->
+
+
+      <aside class="aside">
+        <div
+          class="up-next"
+          style="
+            display: flex;
+            padding-right: 20px;
+            margin-right: 27px;
+            margin-bottom: 5px;
+          "
+        >
+          <p style="margin: 0; margin-right: auto; font-weight: bold;">
+            Up next
+          </p>
+          <p style="margin: 0; padding-right: 5px; font-weight: bold;">
+            Auto play
+          </p>
+          <i class="fa fa-toggle-on" aria-hidden="true"></i>
+        </div>
+
+        <!-- The same video repeated -->
+        <div class="aside-grid">
+          <div class="thumbnail">
+            <a href="#"
+              ><img
+                alt=""
+                style="width: 160px; height: 90px;"
+                src="microverse.jpg"
+            /></a>
+          </div>
+          <div class="thumb-description" style="padding-left: 15px;">
+            <div style="display: flex; justify-content: start;">
+              <h5 style="margin: 0; padding-right: 20px;">
+                From Student to Web Developer:<br />An interview with Juan
+                Manuel..
+              </h5>
+
+              <i class="material-icons">more_vert</i>
+            </div>
+
+            <p style="margin: 5px 0 0 0; padding: 0;">Microverse</p>
+            <p style="margin: 0; padding: 0;">1.9K views . 7 Months ago</p>
+          </div>
```

Comment:
- [ ] I think the fonts-sizes could be closer to the original (channel name, number of views and comments). Make them smaller

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/90047359-2add8b00-dca8-11ea-839f-bae2568e4f15.png) |![image](https://user-images.githubusercontent.com/61479648/90047404-3cbf2e00-dca8-11ea-9dac-0fe9f2ac792b.png) |.